catalog/docs/src/main/resources/_contents/extending-contents.adoc

```diff
@@ -3633,6 +3633,54 @@ Use the Tika Input Transformer for ingesting Microsoft documents, OpenOffice doc
 
 This input transformer maps the metadata common across all mime types to applicable metacard attributes in the default MetacardType.
 
+===== PPTX Input Transformer
+
+The PPTX Input Transformer is input transformer responsible for translating Microsoft PowerPoint (OOXML only) documents into a Catalog Metacard. This input transformer utilizes Apache Tika
```

Comment:
Grammar: `The PPTX Input Transformer is **the** input transformer responsible ... `